lib/script/interpreter.js

```diff
@@ -311,6 +317,52 @@ Interpreter.prototype.evaluate = function() {
   return true;
 };
 
+/**
+ * Checks a locktime parameter with the transaction's locktime
+ *
+ * @param {BN} nLockTime the locktime read from the script
+ * @return {boolean} true if the transaction's locktime is less than or equal to
+ *                   the transaction's locktime
+ */
+Interpreter.prototype.checkLockTime = function(nLockTime) {
```

Comment:
We should likely reference `TransactionSignatureChecker::CheckLockTime` from Bitcoin Core in the comment here, and further information (such as comments?) can be found there.